timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -1010,6 +1124,18 @@ PYBIND11_MODULE(custom_ops, m) {
     declare_nonbonded<double, false>(m, "f64");
     declare_nonbonded<float, false>(m, "f32");
 
+    declare_nonbonded_dense<double, true>(m, "f64_interpolated");
```

Comment:
We should add tests for the pairs and the dense, as this will be used but currently the wrappers aren't tested.

Reply:
Good call. Since this PR is already getting big, I'm thinking to revert exposing `NonbondedDense` and `NonbondedPairs` to user code for now (5d286a5). Then we can test them individually when they're exposed in a future PR.